taker-frontend/src/components/History.tsx

```diff
@@ -62,6 +62,7 @@ interface CfdDetailsProps {
 }
 
 const CfdDetails = ({ cfd, connectedToMaker, displayCloseButton }: CfdDetailsProps) => {
```

Comment:
nit: this could be a separate commit (or at the very least it should be mentioned in the description) - it's a bit unexpected looking at the PR that you're adding the position to all the tiles.